index.html

```diff
@@ -11297,6 +11297,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<ul>
 					<li>Comment: <code>aria-details</code> refers to an element with role <rref>comment</rref>.</li>
 					<li>Definition: <code>aria-details</code> is applied to an element with role <rref>term</rref> and refers to an element with role <rref>definition</rref>.</li>
+					<li>Figure: <code>aria-details</code> is applied to an element with role <rref>figure</rref> and refers to an element with role <rref>caption</rref>, or an element within a <code>caption</code>.</li>
```

Comment:
All the other list items have the referenced element first, so including "Figure" here might make it confusing to scan. Maybe "Caption:" would be better, with the same following text?
```suggestion
					<li>Caption: <code>aria-details</code> is applied to an element with role <rref>figure</rref> and refers to an element with role <rref>caption</rref>, or an element within a <code>caption</code>.</li>
```